app/views/admin/settings/index.html.erb

```diff
@@ -4,7 +4,7 @@
   <% @settings.each do |s| %>
   <p>
     <label for="shinycms_setting_<%= s.id %>"><%= s.name %></label>
-    <%= text_field :settings, "shinycms_setting_#{s.id}", value: s.value %>
+    <%= text_field :settings, "shinycms_setting_#{s.id}", value: s.value, method: :delete %>
```

Comment:
Ooops, move `method: :delete` to the `link_to` below. Relevant route in `config/routes.rb` will need updating too.